lib/utils.js

```diff
@@ -160,32 +165,50 @@ function buildDefaultNotification(
     NotificationTypeVersion: "1",
     Priority: priority,
     Properties: properties,
-    Recipients: recipients.map((recipient) => ({ RecipientId: recipient }))
+    Recipients: recipients?.map(id => {
+      if (cds.env.features.use_global_user_uuid)
+        return { GlobalUserId: id }
+      return { RecipientId: id }
+    })
```

Comment:
this should be more dynamic. As not necessarily all users have a global user id. So I think this can't be solved on a static feature

Reply:
Can you share more details (on teams)? With my naive assumption every solution using IAS+AMS should only have users with a global uuid shouldn't they?